### ISSUE TITLE
Input file formats

### ISSUE THREAD

**Issue by shimbalama** [mgalardini/pyseer#28]
Hi,

Great to see a python version!

Can you please provide example files for phenotypes.tsv and structure.tsv?

Cheers,
Liam

**Comment by mgalardini** [comment]
Hi Liam, thanks for your interest in this implementation of seer.

If you are using pyseer as installed by `pip` or as downloaded from the `master` branch of this repo, the formats are as follows:

- phenotype file:
```6259_5#1	6259_5#1	1
6259_5#2	6259_5#2	1
6259_5#3	6259_5#3	0
6259_5#4	6259_5#4	0
6259_5#5	6259_5#5	0
6259_5#6	6259_5#6	1
6259_5#7	6259_5#7	1
6259_5#8	6259_5#8	1
6259_5#9	6259_5#9	0
6259_5#10	6259_5#10	1
6259_5#12	6259_5#12	1
6259_5#13	6259_5#13	1
6259_5#14	6259_5#14	0
6259_5#15	6259_5#15	0
6259_5#16	6259_5#16	1
6259_5#17	6259_5#17	1
6259_5#18	6259_5#18	1
6259_5#19	6259_5#19	0
6259_5#20	6259_5#20	0
6259_5#21	6259_5#21	0
6259_5#22 6259_5#22 0
[...]
```
Example: https://github.com/mgalardini/pyseer/blob/master/test/subset.pheno
The first column contains the sample names, the last one the phenotypes; there is no need for the second column.

- Population structure file
```
        sample_1        sample_2        sample_3        sample_4
sample_1        0.0     0.0115761       0.0119383       0.0108269
sample_2        0.0115761       0.0     0.0101878       0.0113974
sample_3        0.0119383       0.0101878       0.0     0.0108269
sample_4        0.0108269       0.0113974       0.0108269       0.0
```

Example: https://github.com/mgalardini/pyseer/blob/master/test/distances.tsv.gz
Basically then a square matrix with the distance between each pair of samples, whose name should be the same as the ones included in the phenotypes file.

Please let me know if you need any further assistance, and please bear in mind that the format might change slightly in a future major release of this software.

Best,
Marco

**Comment by shimbalama** [comment]
Thanks so much Marco
